source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -94,7 +94,8 @@ private CalificationsDto GetEvaluatorEvaluation(CalificationsEvaluationDto evalu
             {
                 View = UserView.Evaluation,
                 Evaluation = evaluationDto,
-                Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+                Califications = califications.Where
+                (c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)).ToList()
```

Comment:
Minor styling this: is it common for the predicate to be in a different line, not even with indentation? I'm asking because it's the first time I saw something like that.